src/styles/main.scss

```
@@ -1,3 +1,196 @@
+* {
+  box-sizing: border-box;
+  margin: initial;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+}
+
+.catalog {
+  padding-top: 50px;
+  margin: 0 auto;
+  width: 944px;
+  height: 916px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: space-between;
+}
+
+/* #region navigation */
+.nav__container {
+  text-transform: uppercase;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.nav {
+  background: #fff;
+  background-image: url(/images/logo.png);
+  background-repeat: no-repeat;
+  background-position: 50px center;
+  background-size: 40px;
+
+  display: flex;
+  flex-direction: row-reverse;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__link {
+  margin-right: 20px;
+  position: relative;
+  height: 60px;
+  line-height: 60px;
+  font-weight: 500;
+  text-align: left;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+}
+
+.nav__link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__link:focus {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+  background-color: rgba(0, 172, 220, 1);
+}
+
+.last-link {
   margin: 0;
 }
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
```

```suggestion
.card {
  display: flex;
  flex-direction: column;
  width: 200px;
  padding: 32px 16px 16px;
  border: 1px solid rgba(243, 243, 243, 1);
  border-radius: 5px;
  
  &__image {
    margin: 0 auto 40px;
    width: 160px;
    height: 134px;
    margin-bottom: 40px;  
  }
}
```
Используй вложенность